tools/dune

```diff
@@ -22,6 +22,13 @@
  (modules coqc)
  (libraries coq.toplevel))
 
+(executable
+ (name coqworkmgr)
+ (public_name coqworkmgr)
+ (package coq)
+ (modules coqworkmgr)
```

Comment:
So isn't Dune smart here and able to locate the right module by itself?

Reply:
> So isn't Dune smart here and able to locate the right module by itself?

As I mentioned in other PR the default rule is to use all modules in a dir, that makes sense for a very common setup of one dir, several modules, no library, one executable.